generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.reducer.ts.ejs

```diff
@@ -107,7 +106,7 @@ export default (state: UserManagementState = initialState, action): UserManageme
         ...state,
         updating: false,
         updateSuccess: true,
-        user: action.payload.data
+        user: {}
       };
```

Comment:
@sazeez maybe you can remove all the return statement here as it is the same just the case afte